Gemfile

```diff
@@ -1,8 +1,10 @@
 source "https://rubygems.org"
 
 group :development do
-  gem "test-kitchen", :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
-  gem 'kitchen-vagrant', git: 'https://github.com/test-kitchen/kitchen-vagrant.git', :branch => 'windows-guest-support'
+  gem "test-kitchen"
+  gem 'kitchen-pester'
+  gem 'kitchen-vagrant'
+  gem 'vagrant-wrapper', '>= 2.0'
```

Comment:
Out of curiosity, did you still need vagrant-wrapper with the latest test-kitchen?  I initially thought I needed it but on my Mac, explicitly performing a "gem uninstall vagrant-wrapper" to eliminate the lingering stubs let me run test kitchen without vagrant-wrapper.